pom.xml

```diff
@@ -44,7 +44,7 @@
     <coveralls.maven.plugin.version>4.3.0</coveralls.maven.plugin.version>
     <failsafe.maven.plugin.version>2.22.0</failsafe.maven.plugin.version>
     <findbugs.maven.plugin.version>3.0.5</findbugs.maven.plugin.version>
-    <hotels.oss.plugin.config.version>1.1.0</hotels.oss.plugin.config.version>
+    <hotels.oss.plugin.config.version>1.2.2-SNAPSHOT</hotels.oss.plugin.config.version>
```

Comment:
OK, so in turn the plugin config change is the one to start with and then cascade the release down etc.